python-bindings/Makefile

```diff
@@ -0,0 +1,12 @@
+.PHONY:
```

Comment:
I'd like to have a Python project structure. Something like

```
+ py
  |
  +- carrow (all files go here)
  +- setup.py (project definition)
```

Reply:
What do you want in the root folder, only setup.py ?
what about other files like the runner or pyx pxd